### PR TITLE
Fixed bitwidth not changing in Stepper

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/1395

#### Describe the changes you have made in this PR -
- Fixed the issue
- Changed the default bitwidth of the Stepper from `8` to `1`. Is this fine?

### Sample deployment - https://cvtestexample.herokuapp.com/simulator

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@satu0king @nitin10s Please review!

File: public/js/modules.js [hunk 1]
```diff
@@ -876,12 +876,12 @@ OrGate.prototype.customDraw = function () {
 
 }
 
-function Stepper(x, y, scope = globalScope, dir = "RIGHT") {
+function Stepper(x, y, scope = globalScope, dir = "RIGHT", bitWidth = 1) {
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply public/js/modules.js:879]: This follows standards of other elements. 5 arguments are required.

File: public/js/modules.js [hunk 2]
```diff
@@ -876,12 +876,12 @@ OrGate.prototype.customDraw = function () {
...
 
-    CircuitElement.call(this, x, y, scope, dir, 8);
+    CircuitElement.call(this, x, y, scope, dir, bitWidth);
     this.setDimensions(20, 20);
 
-    this.output1 = new Node(20, 0, 1, this, 8);
+    this.output1 = new Node(20, 0, 1, this);
```
Comment (satu0king) [inline public/js/modules.js:884]: why are you not passing bitwidth here?

**Comment by satu0king** [comment]
Hey @Ananthavijay good attempt. I am glad to see you are peeking into the inner code as well. I made some some changes while merging, take a look at them. 

Do note that we are in production mode. All previous circuits have to work.

1) Default bitwidth 1? why ?  normally its fine but note that all previous circuits assumed default 8, so it won't work! 
2) For some reason you haven't passed bitwidth to the node constructor why ?

**Pull request merged**